tox.ini

```diff
@@ -128,11 +128,12 @@ commands_pre =
   protobuf-1: pip install -r {toxinidir}/opentelemetry-proto/test-requirements-1.txt
 
   getting-started: pip install -r {toxinidir}/docs/getting_started/tests/requirements.txt
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-util-http&subdirectory=util/opentelemetry-util-http
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-instrumentation&subdirectory=opentelemetry-instrumentation
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-instrumentation-requests&subdirectory=instrumentation/opentelemetry-instrumentation-requests
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-instrumentation-wsgi&subdirectory=instrumentation/opentelemetry-instrumentation-wsgi
-  getting-started: pip install -e {env:CONTRIB_REPO}\#egg=opentelemetry-instrumentation-flask&subdirectory=instrumentation/opentelemetry-instrumentation-flask
+; See #3879 and https://pip.pypa.io/en/stable/cli/pip/#exists-action-option
+  getting-started: pip --exists-action i install -e {env:CONTRIB_REPO}\#egg=opentelemetry-util-http&subdirectory=util/opentelemetry-util-http
```

Comment:
If we want this behavior to apply to all these commands, we could also use the PIP_EXISTS_ACTION env var

Reply:
Right, actually, this is a problem in our CI, the fix should be in a workflow file. Testing a new approach.

Reply:
I think this will work but you need to tell tox to pass-through the environment variables https://tox.wiki/en/4.14.2/config.html#pass_env